src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -339,7 +339,8 @@
 									x:Name="Column10"
 									Width="{x:Bind ColumnsViewModel.StatusColumn.Length, Mode=OneWay}"
 									MaxWidth="{x:Bind ColumnsViewModel.StatusColumn.MaxLength, Mode=OneWay}" />
-								<ColumnDefinition x:Name="Column11" Width="0" />
+								<ColumnDefinition Width="Auto" />
+								<ColumnDefinition Width="0" />
```

Comment:
A little confusing because there are a lot of other columns, but why do the others bind to the view model and this one just has a hardcoded width?

Reply:
The column with `width=0` is needed since you can't actually resize the last column of a grid. That column will never have a width different from `0`. It's only purpose is to be there so that we can resize the last useful column (`status`)